README.md

```diff
@@ -1,2 +1,89 @@
-# documentation
-The model documentation
+# De Re CRUD
+#### Never write a CRUD API or maintenance screen again!
+
+If you are a developer building business apps, the odds are that you
+and/or your colleagues spend too much time building and maintaining
+basic
+[CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
+APIs and maintenance screens for all the data your app needs.  The worst
+part is that a good part of it is not even used in your app.  A
+significant portion of the data in most business apps is just stored
+and displayed with any real value added by the software (think of all
+the description fields, notes, instructions, comments, etc).  Your time
+is usually the most expensive resource for software companies.  That
+time should be spent writing software that adds significant value to
+your company and/or your company's customers.
+
```

Comment:
Add a comment here that DeRe CRUD allows you to separate/isolate your business logic from the data.